checks/checkforcontent.go

```diff
@@ -128,7 +128,7 @@ func CheckFilesContent2(shellPathFnPattern string,
 		content, err := c.RepoClient.GetFileContent(file)
 		if err != nil {
 			//nolint
-			return false, sce.Create(sce.ErrScorecardInternal, err.Error())
```

Comment:
repoClient is an interface we control, so its implementation should return a scorecard error already.